### PR TITLE
Starter View Implementation

### PR DESCRIPTION
**Pull request by tlimaye91**
1. I have added a constant file for storing custom colours. 
2. In sceneDelegate file I have changed ContentView to StarterView. 
3. UI implementation of StarterView. 
4. As of now, I have used system fonts. ( I can change it when you will provide the font)
5. As of now, I have crop the "loginlogo" from assets. ( I can change it when you will provide the images).

Please take a look.

### CODE REVIEW

File: Chatkit/Chatkit/SceneDelegate.swift [hunk 1]
```diff
@@ -35,12 +35,12 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
...
         // Add `@Environment(\.managedObjectContext)` in the views that will need the context.
-        let contentView = ContentView().environment(\.managedObjectContext, context)
+        let starterView = StarterView().environment(\.managedObjectContext, context)
 
         // Use a UIHostingController as window root view controller.
         if let windowScene = scene as? UIWindowScene {
             let window = UIWindow(windowScene: windowScene)
-            window.rootViewController = UIHostingController(rootView: contentView)
+            window.rootViewController = UIHostingController(rootView: starterView)
             self.window = window
```
Comment (tlimaye91) [inline Chatkit/Chatkit/SceneDelegate.swift:44]: Changed ContentView to Starter View

**Review by rajhraval — CHANGES_REQUESTED** [review]
- Seems identical to the Asset provided.
- The Font used is Rubik. (Add it)
- Color Hex Value is 2567FF

Then It's good to go.

**Comment by tlimaye91** [comment]
I have updated on the below points - 
1. Seems identical to the Asset provided. 
2. The Font used is Rubik. (Add it)
3. Color Hex Value is 2567FF

**Review by rajhraval — APPROVED** [review]
Looks fine, good to go!

**Review by ygit — APPROVED** [review]
Looks good to merge 👏🎉

**Comment by ygit** [comment]
@tlimaye91 Go ahead, `Squash & Merge` the PR

**Comment by tlimaye91** [comment]
@ygit I don't think I have "write" access to merge.

**Pull request merged**